task_DEV_5/task_DEV_5/BubbleSorter.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.IO;
+using System.Text;
+
+namespace task_DEV_5
+{
+    class BubbleSorter
+    {
+        private string path;
+        private FileStream stream;
+
+        public BubbleSorter(string path)
+        {
+            this.path = path;
+        }
+
+        private string ReadString()
+        {
+            StringBuilder builder = new StringBuilder();
+            char symbol;
+            while ((symbol = (char)stream.ReadByte()) != '\n')
+            {
+                if (stream.Length == stream.Position)
+                {
+                    break;
+                }
+                builder.Append(symbol);
+            }
+            return builder.ToString();
+        }
+
+        private void WritingLinesToAFile(string firstString, string secondString)
+        {
+            byte[] firstStringBytes = Encoding.Default.GetBytes(firstString + '\n');
```

Comment:
Instead of `string + char` better to use `Stringbuilder`.

Reply:
Done.